kqueen/blueprints/api/api2_0.yml

```diff
@@ -0,0 +1,791 @@
+swagger: "2.0"
```

Comment:
Do we really need to have both versions? I know there are some advantages of v 3.0 but is it worth maintaining two versions?

Reply:
3.0 describes the API better, because of the improved format and we should use it after all swagger-api subprojects supports it, but as for now its supported by swagger-ui, but not by swagger-codegen. So we have 3.0 for the future use and current use with swagger-ui and 2.0 for swagger-codegen to potentially generate API clients for the supported languages.